--- is.orig	2010-05-03 09:38:37.000000000 +0000
+++ is	2010-05-03 09:39:01.000000000 +0000
@@ -189,11 +189,13 @@
 
     include "us(dvorak)"
 
-    key <AD11> { [      slash,  question,    U201e,   U201c   ] };
-    key <AC06> { [       d,     D,           eth,     ETH     ] };
-    key <AC10> { [       s,     S,           ae,      AE      ] };
-    key <AC11> { [   minus,     underscore,  endash,  emdash  ] };
-    key <AB10> { [       z,     Z,           thorn,   Thorn   ] };
+    key <AE04> { [	    4,      dollar,      EuroSign,  EuroSign ] };
+    key <AD11> { [      slash,  question,    U201e,   U201c      ] };
+    key <AC06> { [       d,     D,           eth,     ETH        ] };
+    key <AC10> { [       s,     S,           ae,      AE         ] };
+    key <AC11> { [   minus,     underscore,  endash,  emdash     ] };
+    key <AB10> { [       z,     Z,           thorn,   Thorn      ] };
+    key <AB06> { [	     b,	    B,           ssharp,  U1E9E      ] };
 
     include "level3(ralt_switch)"
 };
